### PR TITLE
Fix deprecation on aws_iam_policy_document.source_json in favor of source_policy_documents attr.

### PR DESCRIPTION
**Pull request by azec-pdx**
## what
* I was getting deprecation warning for `data.aws_iam_policy_document.resource.source_json` attribute:
  ```bash
  │ Warning: Argument is deprecated
  │
  │   with module.ecr_public.module.ecr_public.data.aws_iam_policy_document.resource,
  │   on .terraform-devops/modules/ecr_public.ecr_public/main.tf line 43, in data "aws_iam_policy_document" "resource":
  │   43:   source_json = local.principals_full_access_non_empty ? join("", [data.aws_iam_policy_document.resource_full_access[0].json]) : join("", [data.aws_iam_policy_document.empty[0].json])
  │
  │ Use the attribute "source_policy_documents" instead.
  │
  │ (and one more similar warning elsewhere)
  ```

## why
* Simple fix for deprecated attribute, [please see docs here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document#source_json)!

## references
* [Documentation on source_json being deprecated](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document#source_json)

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     ":preserveSemverRanges"
   ],
   "labels": ["auto-update"],
+  "dependencyDashboardAutoclose": true,
```
Comment (korenyoni) [inline .github/renovate.json:7]: Why is this changed?

Comment (azec-pdx) [reply .github/renovate.json:7]: I am not 100% sure, but belongs to commit done by @cloudpossebot : https://github.com/cloudposse/terraform-aws-ecr-public/pull/4/commits/932a629161b70318f05e46ca1ba392adcaaf9ad2

**Review by Gowiem — APPROVED** [review]
Looks solid to me! 

The renovate config change that @korenyoni pointed out was changed by our friendly neighborhood @cloudpossebot so I'm assuming that is just some upstream templating that this repo is getting for the first time and we can safely ignore. Somebody can yell at me if that is not the case 😅

**Comment by korenyoni** [comment]
Yeah didn't check the blame — thanks for the clarification!
